apps/bare-expo/ios/BareExpo.xcodeproj/project.pbxproj

```diff
@@ -455,6 +455,7 @@
 			inputPaths = (
 				"${PODS_ROOT}/Target Support Files/Pods-BareExpoMain-BareExpo/Pods-BareExpoMain-BareExpo-resources.sh",
 				"${PODS_CONFIGURATION_BUILD_DIR}/EXConstants/EXConstants.bundle",
+				"${PODS_CONFIGURATION_BUILD_DIR}/ExpoSQLite/ExpoSQLite.bundle",
```

Comment:
if we removed the resource_bundles from podspec, these changes should be removed.